Sources/JSON.swift

```diff
@@ -0,0 +1,50 @@
+public enum JSON {
+    public enum Number {
+        case integer(Int)
+        case unsignedInteger(UInt)
```

Comment:
Ha, very good point, I haven't considered that. Yup then it makes sense to have this here 👍